Database/Updates/World/06-06-27-2017-generator-chains-testdata.sql

```diff
@@ -0,0 +1,122 @@
+USE `ace_world`;
+
+/* Create new generator link table */
+CREATE TABLE IF NOT EXISTS `ace_object_generator_link` (
+  `aceObjectId` int(10) unsigned NOT NULL,
+  `index` tinyint(3) unsigned NOT NULL,
+  `generatorId` int(10) unsigned NOT NULL,
```

Comment:
`generatorId` column refers to the wcid of the generator to spawn or the generator spawning the aceObjectId? This reads like it also should have a FK cascade delete attached to it so that if either the generator or whatever generator is spawning is deleted, it is removed from the spawn table/links.